services/metadactyl-clj/src/metadactyl/service/apps/de/admin.clj

```diff
@@ -26,43 +25,43 @@
   "Validates the length of an App Category name."
   [name]
   (when (> (count name) max-app-category-name-len)
-    (throw+ {:error_code ce/ERR_ILLEGAL_ARGUMENT
-             :reason     "App Category name too long."
-             :name       name})))
+    (throw+ {:type   :clojure-commons.exception/illegal-argument
+             :error  "App Category name too long."
+             :name name})))
 
 (defn- validate-subcategory-name
   "Validates that the given subcategory name is available under the given App Category parent ID."
   [parent-id name]
   (when (app-groups/category-contains-subcategory? parent-id name)
-    (throw+ {:error_code ce/ERR_ILLEGAL_ARGUMENT
-             :reason     "Parent App Category already contains a subcategory with that name"
-             :parent_id  parent-id
-             :name       name})))
+    (throw+ {:type   :clojure-commons.exception/illegal-argument
+             :error  "Parent App Category already contains a subcategory with that name"
+             :parent_id parent-id
+             :name name})))
```

Comment:
The alignment comment from above applies here too. I prefer to have keys and values aligned in map literals just because it helps me visually distinguish the keys and values. A single space is fine too, though.